docker-compose.yml

```
@@ -0,0 +1,10 @@
+version: '3.9'
+
+services:
+  db:
+    image: postgres
+    restart: always
```

Toda vez que eu reiniciar meu PC esse container safado seu vai iniciar junto. Muda essa política por favor pra evitar isso.
```suggestion
    restart: ???
```

---

Vou atualizar pro on-failure

Por dois motivos:
- Reinicia só se ele der erro.
- O contêiner não reinicia se eu parar manualmente ou se o sistema for reiniciado.

Eu vi que tem como limitar a quantidade de vezes que ele tenta reiniciar. Achei que não seria válido pro meu caso.